examples/braket_features/create_custom_circuit_and_custom_local_simulator.ipynb

```diff
@@ -0,0 +1,1296 @@
+{
```

Comment:
<p>Creating a <code>translations.py</code> file isn't <em>strictly</em> necessary, as it could be done inline. In fact, I think that might be easier to follow; create the translation dicts in the notebook, and use those dicts directly rather than importing them.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/297/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>